src/components/DropdownMenu.js

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import { MdExitToApp, MdChevronRight } from "react-icons/md";
+
+const DropdownMenu = ({ handleLogout, tokenDataEmail }) => {
```

Comment:
looks good, but maybe we could create DropdownItem instead of  to have the button and the option fixed.
`
<li>
                        <button onClick={handleLogout} className="flex items-center w-full p-2 text-left text-sm gap-2 border border-primary hover:text-primary">
                            <div className="rounded-full p-1 bg-slate-400 flex items-center">
                                <MdExitToApp className="text-white" size={24} />
                            </div>
                            <span className="flex-grow" style={{ whiteSpace: 'nowrap' }}>Cerrar Sesión</span>
                            <MdChevronRight className="ml-3" size={24} />
                        </button>
                    </li>
` this could be a component and we can send as parameters the handlelogout and the text or use a for each and recieve an array with the textb and the event